docs/developer/binder.md

```diff
@@ -27,6 +27,116 @@ The scheduler and binder communicate through a custom resource called `BindReque
 
 The BindRequest API serves as a clear contract between the scheduler and binder, allowing them to operate independently.
 
+### Example BindRequest for NVIDIA GPU DRA Driver
```

Comment:
I see you put it under the developers docs.
Dont you prefer to have it in the quickstart examples?

And example of how to run workload with DRA

Reply:
Thanks for adding DRA example, I think it is important enough to have a dedicated section in the docs. Probably in `docs/dra` directory.
Quickstart is a place where we point to users right after installation so they can easily try the very basic flow with KAI Scheduler.

Reply:
Agree. docs/dra sounds good.